src/commands/clearqueue.js

```diff
@@ -0,0 +1,23 @@
+const util = require("../util");
+
+module.exports = {
+    name: "clearqueue",
+    description:"Clears The Total Queue.",
+    aliases: ["clr","clear],
+    exec: (msg) => {
+        const { music } = msg.guild;
+        if (!music.player || !music.player.playing) return msg.channel.send(util.embed().setDescription("❌|  Currently not playing anything."));
+        if (!music.queue.length) return msg.channel.send(util.embed().setDescription("❌ | Queue is empty."));
+
+        if (!msg.member.voice.channel)
+            return msg.channel.send(util.embed().setDescription("❌ | You must be on a voice channel."));
+        if (msg.guild.me.voice.channel && !msg.guild.me.voice.channel.equals(msg.member.voice.channel))
+            return msg.channel.send(util.embed().setDescription(`❌ | You must be on ${msg.guild.me.voice.channel} to use this command.`));
+        try {
+            music.queue.length = [];
```

Comment:
assigning value to Array#length (.queue.length) doesn't change anything, also you can't assign to it because it's not writeable